torchci/test/drciBot.test.ts

```diff
@@ -187,7 +189,9 @@ describe("verify-drci-functionality", () => {
         (url) => url.includes("self/queries"),
         (body) => JSON.stringify(body).includes("issue_comment")
       )
-      .reply(200, { results: [] }); // query to get issue comments
+      .reply(200, { results: [] }) // query to get issue comments
+      .post((url) => url.includes("pr_commits"))
+      .reply(200, { results: [{ sha: "MOCK", message: "Anything goes" }] }); // query to get PR commits from fetchPR
```

Comment:
I'd just like to point out that fetchPR doesn't actually return the commits on the PR, it returns every commit that got pushed that is related to the PR and has a workflow run on it

Reply:
Yeah, I have just read the part Zain added to handle force merge overwriting past commits.